docs/src/pages/[platform]/getting-started/usage/shared/BasicShoppingCard.tsx

```diff
@@ -10,8 +10,8 @@ export const BasicShoppingCard = () => (
     <Image
       alt="Abstract art"
       height="21rem"
+      objectFit="cover"
       src="https://images.unsplash.com/photo-1500462918059-b1a0cb512f1d?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=987"
-      width="100%"
```

Comment:
This fixes and issue where the image renders skinnier than the actual image aspect ratio.